docs/release-notes.asciidoc

```diff
@@ -83,6 +71,8 @@ See ({pull}77327[#77327]) for details.
 [role="screenshot"]
 image::images/detection-rule-failure.png[]
 
+* When adding a rule exception, you can not select value lists with the `ip_range`. Any `ip_range` values will not appear in the dropdown even if you uploaded a value list with `ip_range` ({pull}79511[#79511]).
```

Comment:
I think it would be more accurate if we update to say something like this:
> When adding a rule exception, you cannot select value lists of type `ip_range`. Lists of type `ip_range` will not appear in the Add Exception dropdown as possible values after selecting the `is in list` operator.